utils/ioda_example/gdas_meanioda.h

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include "eckit/config/LocalConfiguration.h"
+#include "ioda/Group.h"
+#include "ioda/ObsSpace.h"
+#include "ioda/ObsVector.h"
+#include "oops/base/PostProcessor.h"
+#include "oops/mpi/mpi.h"
+#include "oops/runs/Application.h"
+#include "oops/util/DateTime.h"
+#include "oops/util/Duration.h"
+#include "oops/util/Logger.h"
+
+namespace gdasapp {
+  // this is an example of how one can use OOPS and IODA to do something
+  // in this code, we will read in configuration from YAML
+  // and then use that configuration to read in a IODA formatted file,
+  // read one group/variable (with optional channel)
+  // and compute and print out the mean of the variable.
+  // Nothing fancy, but you can see how this could be expanded!
+
+  class IodaExample : public oops::Application {
+   public:
+    explicit IodaExample(const eckit::mpi::Comm & comm = oops::mpi::world())
+      : Application(comm) {}
+    static const std::string classname() {return "gdasapp::IodaExample";}
+
+    int execute(const eckit::Configuration & fullConfig, bool /*validate*/) const {
+
+      // get the obs space configuration
+      const eckit::LocalConfiguration obsConfig(fullConfig, "obs space");
+      ioda::ObsTopLevelParameters obsparams;
+      obsparams.validateAndDeserialize(obsConfig);  // TODO CRM, can I remove this and then the simulated vars junk??
+      oops::Log::info() << "obs space: " << std::endl << obsConfig << std::endl;
+      // time window stuff
```

Comment:
how about adding an empty line after all the sequential steps? It would make the code a bit nicer to read.

Reply:
Fixed at https://github.com/NOAA-EMC/GDASApp/pull/553/commits/8bb6a361e73a27883e718a65f1e977bf99632224